src/parsing/compileAstToExpression.ts

```diff
@@ -1064,6 +1067,17 @@ function computedCommentConstructor(ast, compilationOptions) {
 	return new CommentConstructor(expr);
 }
 
+function computedTextConstructor(ast, compilationOptions) {
```

Comment:
No need to type everything is this file, but I think we should make sure everything new is typed.
`function computedTextConstructor(ast: IAST, compilationOptions: CompilationOptions) {`